requirements.txt

```diff
@@ -0,0 +1 @@
+pytest~=6.2.4
```

Comment:
Do we really need pytest in requirements? Currently all requirements are part of `setup.py`. Also pytest is installed explicitly in CI pipeline.